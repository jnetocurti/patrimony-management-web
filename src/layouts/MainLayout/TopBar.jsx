import React from 'react';
import { Link as RouterLink } from 'react-router-dom';

import { AppBar, Toolbar, makeStyles } from '@material-ui/core';

import clsx from 'clsx';
import PropTypes from 'prop-types';
import Logo from 'src/components/Logo';

const useStyles = makeStyles((theme) => ({
  root: {
    backgroundColor: theme.palette.background.default
  },
  toolbar: {
    height: 64
  }
}));

const TopBar = ({ className, ...rest }) => {
  const classes = useStyles();

  return (
    <AppBar className={clsx(classes.root, className)} elevation={0} {...rest}>
      <Toolbar className={classes.toolbar}>
        <RouterLink to="/">
          <Logo />
        </RouterLink>
      </Toolbar>
    </AppBar>
  );
};

TopBar.propTypes = {
  className: PropTypes.string
};

export default TopBar;
