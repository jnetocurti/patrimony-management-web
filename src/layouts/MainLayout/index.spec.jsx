import React from 'react';

import { shallow } from 'enzyme';

import MainLayout from '.';

describe('src/layouts/MainLayout/TopBar', () => {
  it('should render correctly', () => {
    const component = shallow(<MainLayout />);
    expect(component).toMatchSnapshot();
  });
});
