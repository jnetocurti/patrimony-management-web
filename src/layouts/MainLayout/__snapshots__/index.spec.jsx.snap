// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/layouts/MainLayout/TopBar should render correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <TopBar />
  <div
    className="makeStyles-wrapper-2"
  >
    <div
      className="makeStyles-contentContainer-3"
    >
      <div
        className="makeStyles-content-4"
      >
        <Outlet />
      </div>
    </div>
  </div>
</div>
`;
